rules/integrations/aws/exfiltration_dynamodb_scan_by_unusual_user.toml

```diff
@@ -0,0 +1,132 @@
+[metadata]
+creation_date = "2025/03/13"
+integration = ["aws"]
+maturity = "production"
+updated_date = "2025/03/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when an AWS DynamoDB table is scanned by a user who does not typically perform this action. Adversaries may
+use the Scan operation to collect sensitive information or exfiltrate data from DynamoDB tables. This rule detects
+unusual user activity by monitoring for the Scan action in CloudTrail logs. This is a New Terms rule that only flags
+when this behavior is observed by the `aws.cloudtrail.user_identity.arn` for the first time in the last 14 days.
+"""
+false_positives = [
+    """
+    Legitimate users may scan DynamoDB tables for various reasons, such as data analysis or application functionality.
+    Ensure that the user has the necessary permissions and that the Scan operation is authorized before taking action.
+    """,
+]
+from = "now-9m"
+index = ["filebeat-*", "logs-aws.cloudtrail-*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "AWS DynamoDB Scan by Unusual User"
+note = """## Triage and analysis
+
+### Investigating AWS DynamoDB Scan by Unusual User
+
+This rule identifies when an AWS DynamoDB table is scanned by a user who does not typically perform this action. Adversaries may use the Scan operation to collect sensitive information or exfiltrate data from DynamoDB tables. This rule detects unusual user activity by monitoring for the Scan action in CloudTrail logs.
+
+This is a [New Terms](https://www.elastic.co/guide/en/security/current/rules-ui-create.html#create-new-terms-rule) rule that only flags when this behavior is observed for the first time on a host in the last 14 days.
+
+#### Possible Investigation Steps
+
+- **Identify the Actor**: Review the `aws.cloudtrail.user_identity.arn` field to identify the user who requested the subscription. Verify if this actor typically performs such actions and has the necessary permissions. It may be unusual for this activity to originate from certain user types, such as an assumed role or federated user.
```

Comment:
nit: it might be good to try to remove the bold formatting to align as much with the other guides.

Reply:
Will do

Reply:
Fixed.